language/move-lang/src/diagnostics/mod.rs

```diff
@@ -204,18 +204,28 @@ impl Diagnostics {
         self.diagnostics
     }
 
-    #[deprecated]
-    pub fn into_loc_string_vec(self) -> Vec<Vec<(Loc, String)>> {
+    pub fn into_codespan_format(
+        self,
+    ) -> Vec<(
+        codespan_reporting::diagnostic::Severity,
```

Comment:
Nit-pick: should this be `csr::diagnostic::Severity`, since it's imported as such above?

Reply:
This can definitely be fixed in a follow-up or something, no need to go through another round of reviews. Plus I assume you use the full qualified type name in several spots because it was easier to find-and-replace, which is a good idea if so.

Reply:
blargh, yeah I'll make a note to clean it up sneakily in another PR